semantic_release/commit_parser/token.py

```diff
@@ -21,7 +21,7 @@ class ParsedCommit(NamedTuple):
     @property
     def message(self) -> str:
         m = self.commit.message
-        return m.decode("utf-8") if isinstance(m, bytes) else m
+        return m.decode("utf-8") if isinstance(m, bytes) else m.replace("\r", "")
```

Comment:
```suggestion
        message_str =  m.decode("utf-8") if isinstance(m, bytes) else m
        return message_str.replace("\r", "")
```

Could also do it this way

Reply:
This is probably the better way to catch it regardless if it is bytes or not.

Reply:
Yeah... in my tests, it only seemed to be necessary for the string case, but agree with you this is probably better to be consistent and to be sure.